tiledb/sm/query/strategy_base.h

```diff
@@ -99,9 +98,7 @@ class StrategyParams {
   /*                 API               */
   /* ********************************* */
 
-  /**
-   * Accessor for the resources
-   */
+  /** Accessor for the resources. */
```

Comment:
I don't care for single line doc-comments as a rule. I prefer all such comments be expanded. If too many of them look like they're taking up too much space, it's because the documentation is lacking and there should be more multiline comments.